packages/polaris-viz/src/hooks/tests/useWatchActiveSeries.test.tsx

```diff
@@ -0,0 +1,44 @@
+import {mount} from '@shopify/react-testing';
+
+import {useWatchActiveSeries} from '../useWatchActiveSeries';
+
+describe('useWatchActiveSeries', () => {
+  const addEventListenerSpy = jest.fn((_) => {});
+
+  beforeEach(() => {
+    jest
+      .spyOn(window, 'addEventListener')
+      .mockImplementation(addEventListenerSpy);
+  });
```

Comment:
Define these in each test so it's clear what needs to go in to get something specific out